### ISSUE TITLE
NTR, pulmo: Horseshoe lung

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6434]
**Preferred term label:**
horseshoe lung

**Synonyms**

**Definition (free text, please give PubMed ID)**
A horseshoe lung is an uncommon congenital abnormality caused by the partial fusion of the bases of both lungs behind the pericardial sac

Spencer's Pathology of the Lung, edited by Philip Hasleton, and Douglas B. Flieder, Cambridge University Press, 2013. ProQuest Ebook Central, 

**Parent term (use hpo.jax.org/app)**
Abnormal lung lobation HP:0002101

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:185 scimitar syndrome / congenital pulmonary venolobar syndrome (in up to 80% cases)

**Your nano-attribution (ORCID)**
lmu: chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
added
